### ISSUE TITLE
Printing formatted Markdown

### ISSUE THREAD

**Issue by jjg** [Laverna/laverna#250]
Perhaps this already exists but I wasn't able to find it.

I'd like to be able to print a formatted note (as opposed to the markdown source).  I couldn't find a way to bring up a view like this that didn't include things I don't want to print (the list of notes, etc.).

A button or link that opens the formatted note in a new window/tab would be sufficient.

**Comment by Alwaysin** [comment]
Resizing the window ? :D

**Comment by jjg** [comment]
I'm not sure how re-sizing the window helps (it just makes the preview and list smaller :)

Is there a way to view the formatted note w/o the note list or source view on the same page?

**Comment by Alwaysin** [comment]
My bad, misunderstood.

**Comment by GuyFromMars** [comment]
Yes, it would be better if there is a **Print** function.

**Comment by dagoodma** [comment]
I agree with @fest. We need a way to print/export notes as a pdf or something.  Is anyone working on this?  I can take a look into it if not.  Let know.

**Comment by Xat59** [comment]
the printing feature will be great ! :)

**Comment by dagoodma** [comment]
I can look into implementing this in the near future, but I haven't heard much activity from the Laverna developer community as of late, so I'm a bit hesitant.  Does anyone know if there's still development going on, and what the release schedule is like if there's any?

Thanks.

**Comment by kosinsk2** [comment]
Any updates about printing feature?

**Comment by daed** [comment]
@kosinsk2 this repository is basically dead.  I've been continuing work on it (currently at https://github.com/daed/laverna) with the intention of moving it to a new project at https://github.com/connote-team/connote.

I haven't looked at it yet, but I wouldn't think (incoming famous last words) it would be that difficult to set up.  Can you clarify which would be preferable:  A button that directly generates a print dialog, or an export to html/pdf/whatever?

@dagoodma I'm still trying to continue work on laverna so if this is something you're willing to work on almost two years later, I'd be more than happy to take a PR on it at https://github.com/connote-team/connote.
